test/test_nn.py

```diff
@@ -3855,6 +3855,11 @@ def _compatibility_test(sequences, lengths, batch_first):
         self.assertEqual(packed.batch_sizes, [3, 2, 1])
         self.assertEqual(packed.data.data, expected)
 
+        packed = rnn_utils.pack_sequence([a, b, c], batch_first=True)
+        expected = torch.tensor([1, 4, 6, 2, 5, 3])
```

Comment:
This test is insufficient. It would have also passed even if you didn't pass `batch_first=True`. The best example is one that behaves differently depending on `batch_first`.

Reply:
This might be hard to test I think (?) as 
`padded = pad_sequence(sequences, batch_first=batch_first)` will put it in the output format, but then `pack_padded_sequence(padded, [v.size(0) for v in sequences], batch_first=batch_first)` will consume it? I am not too sure how the internal format of PackSequence works